agent/bench-scripts/pbench-trafficgen

```diff
@@ -670,6 +683,16 @@ if [ $one_shot == "y" ]; then
 	cmd="$cmd --one-shot=1"
 fi
 
+# register the tool triggers depending on the tool period preference
+if [ "$tool_period" == "binary-search" ]; then
+	pbench-register-tool-trigger --group=$tool_group --start-trigger="Starting binary-search" --stop-trigger="Finished binary-search"
+else
```

Comment:
Can you consider using `elif` instead to avoid a level of indentation?

Reply:
sure